javaslang/src/main/java/javaslang/collection/HashSet.java

```diff
@@ -559,7 +559,7 @@ public boolean contains(T element) {
             return empty();
         } else {
             final HashArrayMappedTrie<U, U> that = foldLeft(HashArrayMappedTrie.empty(),
-                    (tree, t) -> addAll(tree, mapper.apply(t)));
+                                                            (tree, t) -> addAll(tree, mapper.apply(t)));
```

Comment:
I used provided formatter, and doublechecked it before submitting. Just thought I'll add this byt the way (previous code was not compatible with formatter provided) - should I remove this change ?